runbotics/runbotics-desktop/src/action/excel/python/set_cell_value.py

```diff
@@ -0,0 +1,29 @@
+from RPA.Excel.Files import Files
+import sys
+
+def set_cell_value(path, row, column, value):
+    print(path, row, column, value)
+    if not path:
+        raise ValueError("Path cannot be empty")
+    if not row:
+        raise ValueError("Row cannot be empty")
+    if not column:
+        raise ValueError("Column cannot be empty")
+    if not value:
+        raise ValueError("Value cannot be empty")
+
+    lib = Files()
+    lib.open_workbook(path)
+    try:
+        lib.set_cell_value(row, column, value)
+        lib.save_workbook()
+    finally:
+        lib.close_workbook()
```

Comment:
we need to somehow keep this object instance for later use in other actions

Reply:
so creating the session. already on it!